### DISCUSSION TITLE
拖动节点到画布时显示异常的问题

### DISCUSSION THREAD

**Discussion by TheMonsterSTAN** [didi/LogicFlow#1459] [Q&A]
![image](https://github.com/didi/LogicFlow/assets/30610541/19cc8a51-cd80-4525-910a-61a3626937be)
如上图所示，我在将左侧名为“变量赋值”的节点拖入画布时，同时显示选中了好几个节点，并且鼠标光标是一个禁止的形状。
观察浏览器控制台发现有如下报错：
![image](https://github.com/didi/LogicFlow/assets/30610541/4a4d52dd-810c-4cd5-8100-d492ba8e37f0)
想问下各位大佬，可能是什么原因导致的？

**Comment by boyongjiong** [comment]
你在 issue 里面搜一下？我记得好像有
